query-parser/src/lib.rs

```diff
@@ -8,6 +8,11 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+#![allow(unused_imports)]
+
+#[macro_use(assert_parses_to)]
```

Comment:
Generally I don't see folks listing the macro names here.  I'm fine with not doing so.